apps/vr-tests-react-components/screener.config.js

```diff
@@ -17,26 +17,31 @@ function getCurrentHash() {
 
   return '';
 }
+/**
+ *
+ * @param {string} screenerApiKey
+ * @param {string} sourceBranchName
+ * @param {string} deployUrl
+ * @param {string} targetBranch
+ * @returns
+ */
+function getConfig(screenerApiKey, sourceBranchName, deployUrl, targetBranch) {
+  const baseBranch = targetBranch ? targetBranch.replace(/^refs\/heads\//, '') : 'master';
+  // https://github.com/screener-io/screener-storybook#additional-configuration-options
```

Comment:
it would be great if we can extract this type to encapsulated interface and use it whenever needed